platforms/common/uORB/uORBManager.hpp

```diff
@@ -450,7 +451,7 @@ class uORB::Manager
 
 	static uint8_t orb_get_queue_size(const void *node_handle);
 
-	static bool orb_data_copy(void *node_handle, void *dst, unsigned &generation);
+	static bool orb_data_copy(void *node_handle, void *dst, unsigned &generation, bool update);
```

Comment:
nitpick: the naming could be a bit clearer:
```suggestion
	static bool orb_data_copy(void *node_handle, void *dst, unsigned &generation, bool only_if_updated);
```

Reply:
You are right, I'll do that